src/sb_pointcloud_processing/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sb_pointcloud_processing)
+add_definitions(-std=c++14)
+
+find_package(catkin REQUIRED COMPONENTS 
+    nodelet 
+    roscpp 
+    std_msgs 
+    pcl_ros 
+    pcl_msgs
+    sensor_msgs
```

Comment:
We can simplify this a bit by adding `sb_utils` here, and then removing all references to it elsewhere (it just gets wrapped up in `catkin_INCLUDE_DIRS`, `catkin_LIBRARIES`, etc.